engine/src/main/java/org/hibernate/validator/internal/engine/ValidationContext.java

```diff
@@ -91,9 +91,15 @@
 	private final Object executableReturnValue;
 
 	/**
-	 * The set of already processed unit of works. See {@link ProcessedUnit}.
+	 * The sets of already processed path meta constraint unit of works({@link BeanPathMetaConstraintProcessedUnit})
+	 * {@link }.
```

Comment:
There's something wrong with the javadoc here.

+ I think you can remove "of works" and make unit a plural.